app/models/blog.rb

```diff
@@ -10,7 +10,7 @@ class Blog < ApplicationRecord
   scope :published, -> { where('secret = FALSE') }
 
   scope :search, lambda { |term|
-    where("title LIKE '%#{term}%' OR content LIKE '%#{term}%'")
+    where('title LIKE :term OR content LIKE :term', { term: "%#{sanitize_sql_like(term.to_s)}%" })
```

Comment:
引数の最後にハッシュが来る場合は`{}`を省略することができます。
なのでこういう書き方にすることが多いですね。

```suggestion
    where('title LIKE :term OR content LIKE :term', term: "%#{sanitize_sql_like(term.to_s)}%")
```